financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentScreen.kt

```diff
@@ -141,6 +141,20 @@ private fun ConsentContent(
     }
 }
 
+@Composable
+private fun ConsentLoadingContent() {
+    FinancialConnectionsScaffold(
+        topBar = { FinancialConnectionsTopAppBar(onCloseClick = { }) }
+    ) {
+        Box(
+            modifier = Modifier.fillMaxSize(),
+            contentAlignment = Alignment.Center
+        ) {
```

Comment:
fancy

Reply:
we were showing an ugly corner aligned loading before 👏